R/expect-comparison.R

```diff
@@ -30,7 +46,7 @@ expect_compare <- function(operator = c("<", "<=", ">", ">="), act, exp) {
     ">=" = "not more than"
   )[[operator]]
 
-  cmp <- op(act$val, exp$val)
+  cmp <- op(act$val, exp$val) | eq
```

Comment:
It was necessary to use `|` instead of `||` as the length of `op()` is checked just below, so at this point we cannot assume it is unity.